css/style.css

```diff
@@ -16,4 +10,236 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* GENERAL STYLES */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/* <!-- NAV-BAR --> */
+
+.nav-background {
+  background-color: white;
+  border-bottom: 0.1vw solid #f9f9f9;
+}
+
+.nav-grid {
+  display: grid;
+  grid-template-columns: repeat(2, 50%);
+  padding-top: 0.75rem;
+  padding-bottom: 0.75rem;
+  align-items: center;
+  width: 86%;
+  margin: 0 auto;
+}
+
+.main-nav {
+  display: flex;
+  gap: 6%;
+  justify-content: end;
+  font-size: 0.9rem;
+  width: 98%;
+}
+
+a {
+  text-decoration: none;
+  color: #868e96;
+  font-size: 1.3em;
+  font-weight: 400;
+  transition: color 1.2s;
+}
+
+a:hover,
+a:active {
+  color: #f15b2a;
+  transition: 1s;
+}
+
+.meet-karma {
+  font-weight: bold;
+  color: #495057;
+  font-weight: 500;
+}
+
+/* <!-- MAIN IMAGE AND TEXT --> */
+
+.main-picture {
+  background-image: url("/img/first-background.jpg");
```

Comment:
It is generally advised to add images via HTML instead of CSS. This allows for better accessibility (you can, and should, provide `alt` titles for screen-readers) and improved load times.